server/src/api/modules/competitions/competition.service.js

```diff
@@ -359,7 +360,11 @@ async function edit(id, title, metric, startsAt, endsAt, participants, verificat
     throw new BadRequestError(`The competition has started, the metric cannot be changed.`);
   }
 
-  if (isPast(competition.startsAt) && startsAt && startsAt !== competition.startsAt) {
+  if (
+    startsAt &&
+    isPast(competition.startsAt) &&
+    startDate.toLocaleString() !== competition.startsAt.toLocaleString()
```

Comment:
Any reason you're using string comparisons instead of using timestamps?

Reply:
Not really, didn't notice there was a timestamp option, but I see that's the `getTime()`.
Good point tho, Let me change that 👍

Reply:
@psikoi fixed 😄